tests/convert/coco_expect.json

```diff
@@ -1,1316 +0,0 @@
-{
```

Comment:
Isn't it better to have "real-world examples" for at least integration tests?

Reply:
I think we can break it into small tests, seems to become clearer to me. By now, I just complete the coverage, but need improvements, to make sure that everything is being tested correctly.

Reply:
Until then, I think it's better to keep the old tests...

Reply:
Add an extra test with the real data from old tests on [48ef581](https://github.com/lapix-ufsc/lapixdl/pull/34/commits/48ef581e6a26e7d373a2752c596b7b9fb339ca09)